src/hb-ot-layout-common.hh

```diff
@@ -3002,24 +3254,42 @@ struct FeatureVariations
     return (this+record.substitutions).find_substitute (feature_index);
   }
 
+  void collect_feature_substitutes_with_variations (hb_collect_feature_substitutes_with_var_context_t *c) const
```

Comment:
FYI the behaviour in fontTools maybe changing in the near future: https://github.com/fonttools/fonttools/issues/2737

Don't need to do anything here yet, but we may need to update this in the future.